circuit_knitting_toolbox/entanglement_forging/__init__.py

```diff
@@ -28,6 +28,10 @@
 )
 import circuit_knitting.forging.cholesky_decomposition as cholesky_module
 
+from qiskit_nature import settings
+
+settings.use_pauli_sum_op = False
+
```

Comment:
These lines actually need to be moved to `circuit_knitting/forging/__init__.py` now that #244 is merged.

Reply:
Also, once we finally figure out the magic to get everything working, we should double check that this block is actually required to get tests to pass.  The [migration guide](https://qiskit.org/ecosystem/nature/migration/0.6_c_qubit_converter.html#paulisumop-vs-sparsepauliop) suggests that it is not always necessary to change this setting.  If you provide a `SparsePauliOp` to a function that returns an operator, it should result in a `SparsePauliOp` regardless of this setting, from what I understand.

Reply:
Yes, this was in fact added after a discussion between @caleb-johnson and Max (Rossmannek). He suggested that we can suppress the warnings with these settings until opflow is dropped, which will be in the 0.8.0 release.